### PR TITLE
chore(Cross): [IOAPPX-343,IOAPPX-311] Add `react-native-skia` library + Dynamic header background (under the new Experimental Lab category)

### PR DESCRIPTION
**Pull request by dmnplb**
> [!caution]
> Due to the coexistence of two different versions of `XState`, this PR depends on:
> * https://github.com/pagopa/io-app/pull/6135

This PR restores the original PR, already approved. Read the description to learn more:
* https://github.com/pagopa/io-app/pull/5954

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -16541,7 +16576,7 @@ xss@1.0.10:
     commander "^2.20.3"
     cssfilter "0.0.10"
 
-"xstate5@npm:xstate@5":
+"xstate5@npm:xstate@5", xstate@^5.13.0:
```
Comment (Vangaorth) [inline yarn.lock:16579]: Please remove such changes to xState. After this, it will LGTM

File: yarn.lock [hunk 2]
```diff
@@ -16551,11 +16586,6 @@ xstate@^4.29.0, xstate@^4.33.6:
   resolved "https://registry.yarnpkg.com/xstate/-/xstate-4.35.4.tgz#87b2a45b6c7e84d820f56378408c6531ca5c4662"
   integrity sha512-mqRBYHhljP1xIItI4xnSQNHEv6CKslSM1cOGmvhmxeoDPAZgNbhSUYAL5N6DZIxRfpYY+M+bSm3mUFHD63iuvg==
 
-xstate@^5.13.0:
```
Comment (Vangaorth) [inline yarn.lock:16554]: Please remove such changes to xState. After this, it will LGTM

**Review by Vangaorth — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (Vangaorth) [reply yarn.lock:16579]: @mastro993 I am right?

File: yarn.lock (hunk 2 above)
Comment (Vangaorth) [reply yarn.lock:16554]: @mastro993 I am right?

File: yarn.lock (hunk 1 above)
Comment (dmnplb) [reply yarn.lock:16579]: @mastro993 @Vangaorth Currently, if I remove this line, `yarn` installs two different versions the next time it runs: `5.13.0` and `5.17.4`. If we don't make any changes, there is only one version: `5.13.0`.
I'd leave things as they are.

Comment (mastro993) [reply yarn.lock:16579]: @dmnplb Before such change we must test if the CI builds works correctly and XState machines have the expected behavior.

**Comment by dmnplb** [comment]
@Vangaorth We have solved the problem of the `xstate` dependency, we can go on with the merging of this one as well.

**Review by Vangaorth — APPROVED** [review]

**Pull request merged**